tests/test_from_thread.py

```diff
@@ -628,6 +628,7 @@ async def raise_baseexception() -> None:
             assert exc.value.__context__ is None
 
     @pytest.mark.parametrize("portal_backend_name", get_all_backends())
+    @pytest.mark.usefixtures("deactivate_blockbuster")
```

Comment:
That could also be a marker. LMK what you prefer.

Reply:
Nah, I don't see any real benefit from that.